com.synology.SynologyDrive.yaml

```diff
@@ -0,0 +1,59 @@
+app-id: com.synology.SynologyDrive
+default-branch: stable
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: synology-drive
+tags: [proprietary]
+finish-args:
+  - --filesystem=home
+  - --share=network
+  - --socket=x11
+  - --socket=wayland
+  - --share=ipc
+  - --talk-name=org.kde.StatusNotifierWatcher
+modules:
+  - name: synology-drive
+    buildsystem: simple
+    build-commands:
+      - install -D synology-drive.sh /app/bin/synology-drive
+      - install -Dm644 com.synology.SynologyDrive.desktop -t /app/share/applications
+      - install -Dm644 synologydrive-64.png /app/share/icons/hicolor/64x64/apps/com.synology.SynologyDrive.png
+      - install -Dm644 synologydrive-128.png /app/share/icons/hicolor/128x128/apps/com.synology.SynologyDrive.png
+      - install -Dm644 synologydrive-256.png /app/share/icons/hicolor/256x256/apps/com.synology.SynologyDrive.png
+      - install -D apply_extra -t /app/bin
+      - cp /usr/bin/ar /app/bin
+      - ARCH_TRIPLE=$(gcc --print-multiarch) && cp /usr/lib/${ARCH_TRIPLE}/libbfd-*.so
+        /app/lib
+      - ARCH_TRIPLE=$(gcc --print-multiarch) && ln -s /usr/lib/${ARCH_TRIPLE}/libtinfo.so/app/lib/libtinfo.so.5
+    sources:
+      - type: script
+        dest-filename: apply_extra
+        commands:
+          - ar x synology-drive.deb
+          - tar xf data.tar.xz
+          - mv opt/Synology/SynologyDrive SynologyDrive
+          - rm -r synology-drive.deb control.tar.gz data.tar.xz debian-binary usr
+      - type: file
+        path: synology-drive.sh
+      - type: file
+        path: com.synology.SynologyDrive.desktop
+      - type: file
+        path: icons/synologydrive-64.png
+      - type: file
+        path: icons/synologydrive-128.png
+      - type: file
+        path: icons/synologydrive-256.png
+      - type: extra-data
+        filename: synology-drive.deb
+        only-arches: [x86_64]
+        url: https://global.download.synology.com/download/Tools/SynologyDriveClient/2.0.1-11061/Ubuntu/Installer/x86_64/synology-drive-client-11061.x86_64.deb
+        sha256: c9a55d9debc6e9521b4f465a5dd55aeb7d2eec7d091658a8f40cb360d71d5edf
+        size: 111162324
+      - type: extra-data
+        filename: synology-drive.deb
+        only-arches: [i686]
```

Comment:
the 19.08 runtime no longer supports 32bit x86 applications, you should remove this section.

Reply:
ok

Reply:
For flatpak version just add to demon. (startup programs)

name: Synology Drive Client

command: /usr/bin/flatpak run --branch=stable --arch=x86_64 --command=synology-drive com.synology.SynologyDrive start

Info: Synchronize files in cloud.

Tested on Fedora works. Cheers from Poland.